samples/bluetooth/peripheral_ams_client/README.rst

```diff
@@ -0,0 +1,276 @@
+
+User interface
+**************
+
+LED 1:
+   * Blinks every two seconds, duty cycle 50%, when the main loop is running.
+
+LED 2:
+   * On when connected.
+
+Button 1:
+   * Enable and disable MS track attributes notification.
+
+Button 2:
+   * Request MS to start or pause playback.
+
+Button 3:
+   * Request MS to jump to the next track if it is supported.
+     Otherwise, request MS to turn up the volume.
+
+Button 4:
+   * Request MS to jump to the previous track if it is supported.
+     Otherwise, request MS to turn down the volume.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/bluetooth/peripheral_ams_client`
+
+.. include:: /includes/build_and_run.txt
+
+.. _peripheral_ams_client_testing:
+
+Testing
+=======
+
+After programming the sample to your development kit, you can test it either by connecting to an iOS device or by using `nRF Connect for Desktop`_ that emulates an AMS Server.
+
+Testing with an iOS device
+--------------------------
+
+#. |connect_terminal_specific|
+#. Reset the kit.
+#. Select the device in the iOS :guilabel:`Settings` > :guilabel:`Bluetooth` menu > :guilabel:`Connect`.
+   When requested, proceed with pairing.
+#. Open the :guilabel:`Apple Music` app and bring up a playlist with some songs.
+#. Press **Button 1** to enable song detail updates.
+#. Press **Button 2** to play a song.
+   Press again to pause playback.
+#. Press **Button 3** to jump to the next song.
+#. Press **Button 4** to jump to the previous song.
+
+Testing with nRF Connect for Desktop
+------------------------------------
+
+#. |connect_terminal_specific|
+#. Reset the kit.
+#. Start `nRF Connect for Desktop`_ and select the connected dongle that is used for communication.
+#. Set up the server:
+    a. Select the :guilabel:`Server setup` tab.
+    #. Select :guilabel:`Dongle configuration` > :guilabel:`Load setup`.
+    #. Load the ``AMS_central.ncs`` file located under :file:`samples/bluetooth/peripheral_ams_client` in the |NCS| folder structure.
+#. Click :guilabel:`Apply to device`.
+#. Select the :guilabel:`Connection Map` tab.
+#. Select :guilabel:`Dongle configuration` > :guilabel:`Security parameters`.
+#. Check :guilabel:`Perform Bonding`, and click :guilabel:`Apply`.
+#. Connect to the device from nRF Connect. The device is advertising as "AMS".
+#. Wait until the bond is established.
+   Verify that the UART data is received as follows::
+
+      Connected xx:xx:xx:xx:xx:xx (random)
+      The discovery procedure succeeded
+      Security changed: xx:xx:xx:xx:xx:xx (random) level 2
+      Pairing completed: xx:xx:xx:xx:xx:xx (random), bonded: 1
+
+#. After bonding, verify in nRF Connect that the :guilabel:`Client Characteristic Configuration` (CCCD) value for :guilabel:`Apple Remote Command` and :guilabel:`Apple Entity Update` are set to ``01 00``.
```

Comment:
Not sure whether you can talk about nRF Connect. Maybe it needs to be nRF Connect for Desktop, in every instance.

Reply:
Will modify to nRF Connect for Desktop.